### ISSUE TITLE
Adding a comment doesnt rm -rf the entire database

### ISSUE THREAD

**Issue by LiEnby** [jeff-hykin/JDSL#5]
why was such a crucial feature removed?

**Comment by jeff-hykin** [comment]
Actually that feature should be present. Are you saying that there's a bug? What OS are you running on?

**Comment by LiEnby** [comment]
yeah i tried /* hello world */ but my database is intact, im very disapointed.
also im using TempleOS v5.03

**Comment by jeff-hykin** [comment]
> yeah i tried /* hello world */ but my database is intact, im very disapointed. also im using TempleOS v5.03

Oh there's the confusion. JDSL executes the comments AS code. You'd need to give a code example of deleting the database.

Are you sure you understand the system?

**Comment by Roeezz** [comment]
I've spoken to Tom and we have to let this guy go.

**Comment by DeadEmbed** [comment]
> > yeah i tried /* hello world */ but my database is intact, im very disapointed. also im using TempleOS v5.03
> 
> Oh there's the confusion. JDSL executes the comments AS code. You'd need to give a code example of deleting the database.
> 
> Are you sure you understand the system?

Jake situation?

**Comment by LIZARD-OFFICIAL-77** [comment]
> JDSL executes the comments AS code. 

what could possibly go wrong.
